src/components/Link/__stories__/Link.docs.mdx

```diff
@@ -0,0 +1,62 @@
+import {Source, Props} from '@storybook/addon-docs';
```

Comment:
```suggestion
import {Props} from '@storybook/addon-docs';
import {DocsSource} from '../../../demo/DocsSource/DocsSource';
```
`Source` doesn't support auto switch to dark theme

Reply:
what should I do?
btw Button is same https://github.com/gravity-ui/uikit/blob/ea9e31f39f4ed71bebd708e948f80e034d08e821/src/components/Button/__stories__/Button.docs.mdx?plain=1#L1

Reply:
Use `DocsSource` instead of `Source`. In `Button.docs.mdx` this import is not used.

Reply:
done